docs/data/migration/migration-pickers-v5/migration-pickers-v5.md

```diff
@@ -70,6 +70,21 @@ The `inputFormat` prop has been renamed to `format` on all the pickers component
  />
 ```
 
+### Update expected values in tests
+
+The value rendered in the input has been modified.
+Spaces are added around `/` characters.
+If you are using RTL or if your date contains single digits sections, non-ASCII characters will be added.
```

Comment:
Shouldn't we wait with adding this part of the description?
Regarding the first point: if agree that we are not going to introduce a "breaking fix regression" instead of https://github.com/mui/mui-x/pull/8425 then sure, it makes sense. 👌 

Regarding the single-digit sections: Did we not decide to ditch them? That's why I'd rather say - prioritize implementing it instead of adding a temporarily relevant description. 🤔

Reply:
> For single digits, I did not see any PR on it, and it would look like a breaking change, so I don't think it will be the default behavior.
Yup, that's true, but the most correct way of releasing it IMHO is a debate. Technically, it would be a breaking change, but if done this early in the new version cycle and because it's such an annoying part of testing—we could consider making it default 🤔

Reply:
Yes, for the RTL and single digits tokens cases

Reply:
Indeed, makes sense, but in that case, we should update the start of the paragraph. 🤔 
```diff
- The value rendered in the input has been modified.
+ The value rendered in the input might have been modified.
- Spaces are added around `/` characters.
```